SpaceDuck.UserService/SpaceDuck.KalamburyGame/Controllers/RoomController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SpaceDuck.Common.Models;
+using SpaceDuck.KalamburyGame.Services;
+using System.Threading.Tasks;
+
+namespace SpaceDuck.KalamburyGame.Controllers
+{
+    [Route("kalambury/api/[controller]")]
+    [ApiController]
+    public class RoomController : ControllerBase
+    {
+        private IRoomService roomService;
+        private static GameType GameType = GameType.KalamburyGame;
+
+        public RoomController(IRoomService roomService)
+        {
+            this.roomService = roomService;
+        }
+
+        [Route("all")]
+        [HttpGet]
+        public ActionResult GetRooms()
+        {
+            return Ok(roomService.GetRooms(GameType));
+        }
+
+        [Route("{roomId}")]
+        [HttpGet]
+        public ActionResult GetRoom(int roomId)
+        {
+            return Ok(roomService.GetRoom(roomId));
+        }
+
+        [HttpPost]
+        public async Task<ActionResult> CreateRoom(RoomConfiguration roomConfiguration)
+        {
+            var room = roomService.CreateRoom(roomConfiguration, GameType);
+
+            await roomService.SetRoom(room);
+
+            return Ok();
+        }
+
+        [Route("{roomId}/{playerId}")]
+        [HttpPost]
+        public async Task<ActionResult> AddPlayerToRoom(int roomId, string playerId)
+        {
+            var result = await roomService.AddPlayerToRoom(roomId, playerId);
+
+            var message = result ? $"Add to room: {roomId}" : $"Can not add to room: {roomId}";
```

Comment:
Taka uwaga stylistyczna, skoro on się już dodał, to nie lepiej added?
I tak samo removed?

Reply:
spoko powinno być -ed